### PR TITLE
release: Kata Containers 1.11.0

### PR DESCRIPTION
**Pull request by jcvenegas**
- clh: backport: Move to v0.7.0
- backport 1.11: fix podman vfio

2d251652 clh: vsock: Supply the right VsockConfig to Vmconfig
fcc9e93b versions: Move to cloud-hypervisor v0.7.0
34be9e0c clh: memory: remove pmem size argument
9798e8a5 versions: Move to latest cloud-hypervisor
93b1b833 virtcontainers: constrain runtime after creating network
fc9be990 virtcontainers: update sandbox's device cgroup
5cfae217 virtcontainers: remove all the code related to HasCRIContainerType
cff5392a virtcontainers: apply constraints to the sandbox cgroup
ce6edc5a pkg/cgroups: update the list of devices for the hypervisor
3fceece3 pkg/cgroups: add methods to add and remove device from the cgroup
b3458550 pkg/cgroups: implement functions to get information from a host device
0d3b6975 device: add GetHostPath() to generic device

Signed-off-by: katacontainersbot <katacontainersbot@gmail.com>

### CODE REVIEW

**Comment by jcvenegas** [comment]
Keeps failing kata-env without any useful information
```
00:25:20.154 Logging kata-env information:
00:25:20.772 Build step 'Execute shell' marked build as failure
00:25:29.023 Performing Post build task...
```

**Comment by jcvenegas** [comment]
I have skip the run of kata-env than unit test find an panic

```
panic: runtime error: index out of range [0] with length 0 [recovered]
	panic: runtime error: index out of range [0] with length 0

goroutine 228 [running]:
testing.tRunner.func1(0xc0003fc200)
	/usr/local/go/src/testing/testing.go:874 +0x69f
panic(0x16a2560, 0xc000511f80)
	/usr/local/go/src/runtime/panic.go:679 +0x1b2
github.com/kata-containers/runtime/cli.constructVersionInfo(0x173a00f, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/tmp/jenkins/workspace/kata-containers-runtime-ubuntu-18-04-PR/go/src/github.com/kata-containers/runtime/cli/utils.go:164 +0x53a
github.com/kata-containers/runtime/cli.getExpectedRuntimeDetails(0x1738ad8, 0x4, 0x400000001, 0x100000800, 0xa00002000, 0x0, 0xc0003fc300, 0x6, 0x8, 0x0, ...)
	/tmp/jenkins/workspace/kata-containers-runtime-ubuntu-18-04-PR/go/src/github.com/kata-containers/runtime/cli/kata-env_test.go:359 +0xd2
github.com/kata-containers/runtime/cli.getExpectedSettings(0x1738ad8, 0x4, 0x400000001, 0x100000800, 0xa00002000, 0x0, 0xc0003fc300, 0x6, 0x8, 0x0, ...)
	/tmp/jenkins/workspace/kata-containers-runtime-ubuntu-18-04-PR/go/src/github.com/kata-containers/runtime/cli/kata-env_test.go:379 +0xc6
github.com/kata-containers/runtime/cli.TestEnvGetEnvInfoSetsCPUType(0xc0003fc200)
	/tmp/jenkins/workspace/kata-containers-runtime-ubuntu-18-04-PR/go/src/github.com/kata-containers/runtime/cli/kata-env_amd64_test.go:47 +0x43b
testing.tRunner(0xc0003fc200, 0x1793470)
	/usr/local/go/src/testing/testing.go:909 +0x19a
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:960 +0x652
FAIL	github.com/kata-containers/runtime/cli	7.141s
``` 
I fixed in ths PR, I am not sure what caused the panic.

Now the kata-env command worked and this is the output
```
01:22:14.708   Version = "1.0.24"
01:22:14.708 
01:22:14.708 [Runtime]
01:22:14.708   Debug = false
01:22:14.708   Trace = false
01:22:14.708   DisableGuestSeccomp = true
01:22:14.708   DisableNewNetNs = false
01:22:14.708   SandboxCgroupOnly = false
01:22:14.708   Path = "/usr/local/bin/kata-runtime"
01:22:14.708   [Runtime.Version]
01:22:14.708     OCI = "1.0.1-dev"
01:22:14.708     [Runtime.Version.Version]
01:22:14.708       Semver = "1.11.0"
01:22:14.708       Major = 1
01:22:14.708       Minor = 11
01:22:14.708       Patch = 0
01:22:14.708       Commit = "323aeee080a612eb2616f04287898f1a83c04c6a"
01:22:14.708   [Runtime.Config]
01:22:14.708     Path = "/usr/share/defaults/kata-containers/configuration.toml"
01:22:14.708 
01:22:14.708 [Hypervisor]
01:22:14.708   MachineType = "pc"
01:22:14.708   Version = "QEMU emulator version 4.1.1 (kata-static)\nCopyright (c) 2003-2019 Fabrice Bellard and the QEMU Project developers"
01:22:14.708   Path = "/usr/bin/qemu-system-x86_64"
01:22:14.708   BlockDeviceDriver = "virtio-scsi"
01:22:14.708   EntropySource = "/dev/urandom"
01:22:14.708   SharedFS = "virtio-9p"
01:22:14.708   VirtioFSDaemon = "/usr/bin/virtiofsd"
01:22:14.708   Msize9p = 8192
01:22:14.708   MemorySlots = 10
01:22:14.708   PCIeRootPort = 0
01:22:14.708   HotplugVFIOOnRootBus = false
01:22:14.708   Debug = false
01:22:14.708   UseVSock = false
01:22:14.708 
01:22:14.708 [Image]
01:22:14.708   Path = "/usr/share/kata-containers/kata-containers-clearlinux-33050-osbuilder-e55080d-agent-80de213.img"
01:22:14.708 
01:22:14.708 [Kernel]
01:22:14.708   Path = "/usr/share/kata-containers/vmlinuz-5.4.32-73"
01:22:14.708   Parameters = "systemd.unit=kata-containers.target systemd.mask=systemd-networkd.service systemd.mask=systemd-networkd.socket scsi_mod.scan=none agent.log=debug"
01:22:14.708 
01:22:14.708 [Initrd]
01:22:14.708   Path = ""
01:22:14.708 
01:22:14.708 [Proxy]
01:22:14.708   Type = "kataProxy"
01:22:14.708   Path = "/usr/libexec/kata-containers/kata-proxy"
01:22:14.708   Debug = false
01:22:14.708   [Proxy.Version]
01:22:14.708     Semver = "1.11.0-a70f47833c8cc9e2c9cae92c03ee5db0b1c88e73"
01:22:14.708     Major = 1
01:22:14.708     Minor = 11
01:22:14.708     Patch = 0
01:22:14.708     Commit = "<<unknown>>"
01:22:14.708 
01:22:14.708 [Shim]
01:22:14.708   Type = "kataShim"
01:22:14.708   Path = "/usr/libexec/kata-containers/kata-shim"
01:22:14.708   Debug = false
01:22:14.708   [Shim.Version]
01:22:14.708     Semver = "1.11.0-a205eeab86a7635ad761cf9254de0ebaece8ded4"
01:22:14.708     Major = 1
01:22:14.708     Minor = 11
01:22:14.708     Patch = 0
01:22:14.708     Commit = "<<unknown>>"
01:22:14.708 
01:22:14.708 [Agent]
01:22:14.708   Type = "kata"
01:22:14.708   Debug = false
01:22:14.708   Trace = false
01:22:14.708   TraceMode = ""
01:22:14.708   TraceType = ""
01:22:14.708 
01:22:14.708 [Host]
01:22:14.708   Kernel = "5.0.0-1028-azure"
01:22:14.708   Architecture = "amd64"
01:22:14.708   VMContainerCapable = true
01:22:14.708   SupportVSocks = true
01:22:14.708   [Host.Distro]
01:22:14.708     Name = "Ubuntu"
01:22:14.708     Version = "18.04"
01:22:14.708   [Host.CPU]
01:22:14.708     Vendor = "GenuineIntel"
01:22:14.708     Model = "Intel(R) Xeon(R) Platinum 8171M CPU @ 2.60GHz"
01:22:14.708 
01:22:14.708 [Netmon]
01:22:14.708   Path = "/usr/libexec/kata-containers/kata-netmon"
01:22:14.708   Debug = false
01:22:14.708   Enable = false
01:22:14.708   [Netmon.Version]
01:22:14.708     Semver = "1.11.0"
01:22:14.708     Major = 1
01:22:14.708     Minor = 11
01:22:14.708     Patch = 0
01:22:14.708     Commit = "<<unknown>>"'
```

File: cli/utils.go [hunk 1]
```diff
@@ -170,7 +176,7 @@ func constructVersionInfo(version string) VersionInfo {
 			Major:  sv.Major,
 			Minor:  sv.Minor,
 			Patch:  sv.Patch,
-			Commit: pres[1],
+			Commit: pres,
```
Comment (amshinde) [inline cli/utils.go:179]: This seems the likely cause of failure. Maybe the commit is missing for some reason?

**Pull request merged**